featuretools/entityset/entityset.py

```diff
@@ -143,38 +143,44 @@ def metadata(self):
     def reset_data_description(self):
         self._data_description = None
 
-    def to_pickle(self, path, compression=None):
-        '''Write entityset to disk in the pickle format, location specified by `path`.
+    def to_pickle(self, path, compression=None, profile_name=None):
+        '''Write entityset in the pickle format, location specified by `path`.
+            Path could be a local path or a S3 path (will serialize as a tar archive).
```

Comment:
This could be interpreted as saying that tar archives are created locally as well.